themes/ddbasic/sass/base/form.scss

```diff
@@ -296,6 +296,14 @@ input[type="radio"] {
     right: 0;
     top: 0;
   }
+
+  &.multiple {
```

Comment:
`multiple` is very generic. In the spirit of improved CSS I think this should be `select-wrapper-multiple`.